.circleci/config.yml

```diff
@@ -193,13 +189,34 @@ commands:
       study_key:
         type: string
         default: UNKNOWN
+      study_keys:
+        type: string
+        default: *study_keys
+      study_guids:
+        type: string
+        default: *study_guids
     steps:
       - run:
           name: Set environment variables to build <<parameters.study_key>>
           command: |
             echo "export VAULT_TOKEN=`vault write -field=token auth/approle/login role_id=$VAULT_ROLE_ID secret_id=$VAULT_ROLE_SECRET_ID`" >> $BASH_ENV
             echo 'export PATH=$PATH:~/repo/build-utils' >> $BASH_ENV
             if [[ '<<parameters.study_key>>' != 'UNKNOWN' ]]; then
+              echo 'export STUDY_KEY=<<parameters.study_key>>' >> $BASH_ENV
+              STUDY_KEYS=(<<parameters.study_keys>>)
+              STUDY_GUIDS=(<<parameters.study_guids>>)
+              for i in ${!STUDY_KEYS[@]}; do
+                  currentKey=${STUDY_KEYS[$i]}
+                  if [[ $currentKey == '<<parameters.study_key>>' ]]; then
+                      STUDY_GUID="${STUDY_GUIDS[$i]}"
+                      echo "export STUDY_GUID=${STUDY_GUID}" >> $BASH_ENV
+                      break;
+                  fi
+              done;
```

Comment:
This is where we figure out the STUDY_GUID. Bash makes it a bit of a pain to work with arrays.